From 2c313f4b07badb3fa400ef5161bc119f3e7b361f Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Thu, 26 Sep 2019 12:31:15 -0500
Subject: [PATCH 39/45] boot: efi: comment out success validation message

When booting with sd-boot the message "SHA256 validated" is displayed,
this is useful when debugging, but for end user could cause uncertainty.

Let's do not print success validation message just error messages by
comment this print function out.

Signed-off-by: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Signed-off-by: Auke Kok <auke-jan.h.kok@intel.com>
---
 src/boot/efi/random-seed.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/boot/efi/random-seed.c b/src/boot/efi/random-seed.c
index 6de520c..150a2d2 100644
--- a/src/boot/efi/random-seed.c
+++ b/src/boot/efi/random-seed.c
@@ -215,7 +215,7 @@ static VOID validate_sha256(void) {
                 }
         }
 
-        Print(L"SHA256 validated\n");
+        /* Print(L"SHA256 validated\n"); */
 #endif
 }
 
-- 
2.31.1

